frontend/.eslintrc.js

```diff
@@ -16,7 +16,8 @@ module.exports = {
   plugins: ['react'],
   rules: {
     'no-unused-vars': ['warn', { varsIgnorePattern: '^_' }],
-    '@typescript-eslint/no-unused-vars': ['warn', { varsIgnorePattern: '^_' }]
+    '@typescript-eslint/no-unused-vars': ['warn', { varsIgnorePattern: '^_' }],
+    'react/react-in-jsx-scope': ['off'],
```

Comment:
We don't need this rule since NextJS does magic or something so React is always in scope.

Reply:
Oh, nice!